portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,22 +27,43 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-    String convertTojson(ArrayList<String> comments) {
-        Gson gson = new Gson();
-        String convertedTojson = gson.toJson(comments);
-        return convertedTojson;
-    }
+  private String comments = "";
+  private ArrayList<String> messages = new ArrayList<>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    ArrayList<String> messages = new ArrayList<>();
 
-    messages.add(0, "Star Wars is awesome!");
-    messages.add(1, "I will become the next UN secretary general.");
-    messages.add(2, "I am to go back to school.");
-
-    String comments = messages.get((int) (Math.random() * messages.size()));
     response.setContentType("text/html;");
     response.getWriter().println(comments);
   }
+  /*
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      String statement = comment(request, "texts");
+      messages.add(statement);
+
+      addcomment(messages, comments, statement);
+
+      response.sendRedirect("/comments.html");
+  }
+
+  private String comment(HttpServletRequest request, String message) {
+    String comment = request.getParameter(message);
+    return fname;
+  }
+
+  private void addcomment(ArrayList<String> msglist, \
+  String statementlist, String statements) {
```

Comment:
Consider your naming schema. statementlist feels like it should be `statementList` to be in line with your other comments and the style guide. similarly consider this for msglist - should it be msgList? or messageList? If this function got really complicated what would make it easier to search/parse for in the future?